pkg/graveler/committed/apply_test.go

```diff
@@ -59,19 +58,19 @@ func TestApplyReplace(t *testing.T) {
 	ctrl := gomock.NewController(t)
 	defer ctrl.Finish()
 
-	range2 := &committed.Range{ID: "two", MaxKey: committed.Key("dz")}
+	range2 := &committed.Range{ID: "two", MaxKey: committed.Key("dz"), Count: 1, EstimatedSize: 1020}
```

Comment:
Because this range is added as a range, on l. 37 it's part of overlapping ranges which causes the keys written and not the range. I am adding metadata to all the ranges in this file in order to make it less confusing. Thanks